src/safeds/ml/classical/regression/_baseline_regressor.py

```diff
@@ -0,0 +1,199 @@
+import copy
+from concurrent.futures import ALL_COMPLETED, wait
+from typing import Self
+
+from safeds._validation._check_columns_are_numeric import _check_columns_are_numeric
+from safeds.data.labeled.containers import TabularDataset
+from safeds.exceptions import (
+    DatasetMissesDataError,
+    DatasetMissesTargetError,
+    FeatureDataMismatchError,
+    ModelNotFittedError,
+)
+from safeds.ml.classical.regression import (
+    AdaBoostRegressor,
+    DecisionTreeRegressor,
+    ElasticNetRegressor,
+    GradientBoostingRegressor,
+    LassoRegressor,
+    LinearRegressor,
+    RandomForestRegressor,
+    Regressor,
+    RidgeRegressor,
+    SupportVectorRegressor,
+)
+
+
+def _fit_single_model(model: Regressor, train_data: TabularDataset) -> Regressor:
+    return model.fit(train_data)  # pragma: no cover
+
+
+def _predict_single_model(model: Regressor, test_data: TabularDataset) -> TabularDataset:
+    return model.predict(test_data)  # pragma: no cover
+
```

Comment:
why are they not tested?

Reply:
They are being implicitly tested, as they are called a bunch of times in the other Baseline tests. Codecov doesn't seem to recognize this, probably because we are starting new processes and calling them there.